### PR TITLE
[devtools-translate] Russian - What's New in DevTools (Chrome 96)

### LINKED ISSUE #1619: [devtools-translate] Russian - What's New in DevTools (Chrome 96)
Post: https://developer.chrome.com/blog/new-in-devtools-96/

Translator: @solarrust
Reviewers: @webmaxru, @kateryna-prokopenko

Stable rollout: Nov 16
Translation ready by: Nov 7

### PR DESCRIPTION
**Pull request by solarrust**
Fixes #1619 

@webmaxru @kateryna-prokopenko welcome to review! 🙌

### CODE REVIEW

**Comment by solarrust** [comment]
Hi @webmaxru @kateryna-prokopenko !
How can I help you with review of my translation?

File: site/ru/blog/new-in-devtools-96/index.md [hunk 1]
```diff
@@ -29,170 +28,215 @@ draft: true
...
 
-Задачи в трекере Chromium: [1254259](https://crbug.com/1254259)
+Задача в трекере Chromium: [1254259](https://crbug.com/1254259)
 
 
 <!-- ### Retain the Console sidebar {: #console-sidebar } -->
+### Сохранение боковой колонки в Консоли {: #console-sidebar }
 
 <!-- The Console sidebar is here to stay. In Chrome 94, we announced the [upcoming deprecation of the Console sidebar](/blog/new-in-devtools-94/#deprecated) and ask developers for feedback and concerns. -->
+Боковую колонку в Консоли (Console) решено оставить. В Chrome 94 мы анонсировали [упразднение боковой колонки в Консоли](/blog/new-in-devtools-94/#deprecated) и попросили разработчиков высказать свои размышления и сомнения по этому поводу.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-96/index.md:192]:
```suggestion
Боковая колонка в Консоли (Console) остается. В Chrome 94 мы анонсировали предстоящее [упразднение боковой колонки в Консоли](/blog/new-in-devtools-94/#deprecated) и попросили разработчиков высказать свои размышления и сомнения по этому поводу.
```

File: site/ru/blog/new-in-devtools-96/index.md [hunk 2]
```diff
@@ -29,170 +28,215 @@ draft: true
...
 
 <!-- We have now got enough feedback from the deprecation notice and we will work on improving the sidebar rather than removing it. -->
+Мы получили достаточное количество отзывов, и теперь мы будем работать над улучшением боковой панели, а не над её удалением.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-96/index.md:195]:
```suggestion
Мы теперь получили достаточное количество отзывов, и будем работать над улучшением боковой панели, а не над её удалением.
```

File: site/ru/blog/new-in-devtools-96/index.md [hunk 3]
```diff
@@ -29,170 +28,215 @@ draft: true
...
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/b0650096c934bf60c21d51ae8a51c94e8f907d38 #}
 
 Задачи в трекере Chromium: [1232937](https://crbug.com/1232937), [1255586](https://crbug.com/1255586)
 
 
 <!-- ## Deprecated Application cache pane in the Application panel {: #app-cache } -->
+## Устаревшая панель Кэш приложения на вкладке Приложение {: #app-cache }
 
 <!-- The [Application cache](https://developer.chrome.com/docs/devtools/storage/applicationcache/) pane in the Application panel is now removed as the support for [AppCache](https://web.dev/appcache-removal/) is removed from Chrome and other Chromium-based browsers. -->
+Панель [Кэш приложения](https://developer.chrome.com/docs/devtools/storage/applicationcache/) на вкладке Приложение удалена поскольку [AppCache](https://web.dev/appcache-removal/) больше не поддерживается в Chrome и других браузеров на основе Chromium.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-96/index.md:208]:
```suggestion
Панель [Кэш приложения](https://developer.chrome.com/docs/devtools/storage/applicationcache/) (Application cache) на вкладке Приложение (Application) удалена, поскольку [AppCache](https://web.dev/appcache-removal/) больше не поддерживается в Chrome и других браузеров на основе Chromium.
```

File: site/ru/blog/new-in-devtools-96/index.md [hunk 4]
```diff
@@ -29,170 +28,215 @@ draft: true
...
 {% Aside %}
 <!-- To enable the experiment, check the **Enable Reporting API panel in the Application panel** checkbox under **Settings** > **Experiments**. -->
+Чтобы включить эксперимент поставьте галочку в чекбоксе напротив пункта **Enable Reporting API panel in the Application panel** во вкладке **Настройки** (Settings) > **Экспериментальные функции** (Experiments).
 {% endAside %}
 
 <!-- The [Reporting API](https://web.dev/reporting-api/) is designed to help you monitor security violations of your page, deprecated API calls, and more.  -->
+[Reporting API](https://web.dev/reporting-api/) призван помочь вам отслеживать нарушения безопасности вашей страницы, устаревшие вызовы API и многое другое. 
 
 <!-- With this experiment enabled, you can now view the reports status in the new **Reporting API** pane in the **Application** panel.  -->
+После включения этого эксперимента вы сможете просматривать отчёты в панели **Reporting API** во вкладке **Приложение** (Application).
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-96/index.md:227]:
```suggestion
С включенным экспериментом вы теперь можете просматривать статус отчётов в панели **Reporting API** во вкладке **Приложение** (Application). 
```

File: site/ru/blog/new-in-devtools-96/index.md [hunk 5]
```diff
@@ -29,170 +28,215 @@ draft: true
...
+### Эмуляция CSS-медиафичи prefers-contrast {: #prefers-contrast }
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/47fsHvVLiVC9J0eWY9wD.png", alt="Emulate the CSS prefers-contrast media feature", width="800", height="483" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/47fsHvVLiVC9J0eWY9wD.png", alt="Эмуляция CSS-медиафичи prefers-contrast", width="800", height="483" %}
 
 <!-- The [prefers-contrast](https://www.chromestatus.com/feature/5646323212615680) media feature is used to detect if the user has requested more or less contrast in the page. -->
+Медиафича [prefers-contrast](https://www.chromestatus.com/feature/5646323212615680) используется для определения того, запросил ли пользователь больший или меньший кнотраст на странице.
 
 <!-- Open the [Command Menu](/docs/devtools/command-menu/), run the **Show Rendering** command, and then set the **Emulate CSS media feature prefers-contrast** dropdown. -->
+Откройте [Командное меню (Command Menu)](/docs/devtools/command-menu/), выполните команду **Показать "Отрисовка"** (Show Rendering) и выберите нужное значение в выпадающем меню **Эмулировать медиафункцию CSS prefers-contrast** (Emulate CSS media feature prefers-contrast).
```
[the remark this answers is not in the document]
Comment (solarrust) [reply site/ru/blog/new-in-devtools-96/index.md:67]: I took translation from Canary DevTools

CleanShot 2021-11-10 at 14 02 26

And «медиафункция» in Russian shouldn't have a dash =)

File: site/ru/blog/new-in-devtools-96/index.md [hunk 6]
```diff
@@ -29,170 +28,215 @@ draft: true
...
 
-Задачи в трекере Chromium: [1251176](https://crbug.com/1251176)
+Задача в трекере Chromium: [1251176](https://crbug.com/1251176)
 
 
 <!-- ### Proper `Intl` objects preview and evaluation in the Console {: #intl } -->
+### Правильный предварительный просмотр и оценка объектов `Intl` в Консоли {: #intl }
 
 <!-- The [Intl](https://tc39.es/ecma402/#intl-object) objects have proper preview now and are evaluated eagerly in the Console. Previously, the `Intl` objects were not evaluated eagerly. -->
+Объекты [Intl](https://tc39.es/ecma402/#intl-object) теперь имеют удобный предварительный просмотр и их легко оценить в Консоли (Console). Ранее объекты `Intl` не отображались надлежащим образом.
```
[the remark this answers is not in the document]
Comment (solarrust) [reply site/ru/blog/new-in-devtools-96/index.md:165]: What is «энергичные вычисления»? =)

File: site/ru/blog/new-in-devtools-96/index.md (hunk 2 above)
Comment (solarrust) [reply site/ru/blog/new-in-devtools-96/index.md:195]: It's strange sentence «Мы теперь получили» 🤔

**Review by solarrust — COMMENTED** [review]
Thank you for your time, @kateryna-prokopenko 🙌

I add some comments and questions.

Can you, please, explain me when I should use «Панель» and when «Вкладка»? I have no clear understanding of this right now.

File: site/_includes/partials/devtools/ru/whats-new.md [hunk 7]
```diff
@@ -3,18 +3,20 @@
...
-* [New Payload tab in the Network panel](/blog/new-in-devtools-96/#payload)
-* [Improved the display of properties in the Properties pane](/blog/new-in-devtools-96/#properties)
-* [Option to hide CORS errors in the Console](/blog/new-in-devtools-96/#hide-cors-errors)
-* [Proper `Intl` objects preview and evaluation in the Console](/blog/new-in-devtools-96/#intl)
-* [Consistent async stack traces](/blog/new-in-devtools-96/#async)
-* [Retain the Console sidebar](/blog/new-in-devtools-96/#console-sidebar)
-* [[Experimental] New Reporting API pane in the Application panel](/blog/new-in-devtools-96/#reporting-api) -->
+### Chrome 96 {: #chrome96 }
+* [Ранний доступ: Новая вкладка Обзор CSS](/ru/blog/new-in-devtools-96/#css-overview)
+<!-- * [Restored and improved CSS length edit and copy experince](/ru/blog/new-in-devtools-966/#length) -->
```
Comment (chybie) [inline site/_includes/partials/devtools/ru/whats-new.md:8]: Please update this

File: site/ru/blog/new-in-devtools-96/index.md [hunk 8]
```diff
@@ -29,170 +28,235 @@ draft: true
...
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="CSS Overview panel", width="800", height="509" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/fXXPihV3bTl82WDJGX51.png", alt="Вкладка Обзор CSS", width="800", height="509" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/ef26abc89035075bbdb08f1b26c1b8fd942ffc04 #}
 
-Задачи в трекере Chromium: [1254557](https://crbug.com/1254557)
+Задача в трекере Chromium: [1254557](https://crbug.com/1254557)
+
+
+<!-- ## Restored and improved CSS length edit and copy experince {: #length } -->
```
Comment (chybie) [inline site/ru/blog/new-in-devtools-96/index.md:55]: I added a new section here as we merged some fixes in M96 after my article is published. Please help to translate as well. Thanks!

**Review by chybie — CHANGES_REQUESTED** [review]

File: site/ru/blog/new-in-devtools-96/index.md (hunk 2 above)
Comment (kateryna-prokopenko) [reply site/ru/blog/new-in-devtools-96/index.md:215]: It's fine to leave it out altogether

**Comment by kateryna-prokopenko** [comment]
https://ru.wikipedia.org/wiki/%D0%A1%D1%82%D1%80%D0%B0%D1%82%D0%B5%D0%B3%D0%B8%D1%8F_%D0%B2%D1%8B%D1%87%D0%B8%D1%81%D0%BB%D0%B5%D0%BD%D0%B8%D1%8F
You can find энергичные вычисления used for eager evaluation here, if you search the article.

As for tab, panel and pane, the first two (tab and panel) should be translated as вкладка and pane as панель.

Thanks!

**Comment by solarrust** [comment]
@kateryna-prokopenko cool, thank you for the link and for explanation about tabs and panels 🙌

**Comment by solarrust** [comment]
@chybie I added translation of the new block ✅

**Comment by chybie** [comment]
Thanks Alena! I've ping webmaxru for final review. :)

File: site/ru/blog/new-in-devtools-96/index.md [hunk 9]
```diff
@@ -29,170 +28,243 @@ draft: true
...
 
 
 <!-- ## Console updates -->
+## Обновления Консоли
 
 <!-- ### Option to hide CORS errors in the Console {: #hide-cors-errors } -->
+### Опция для скрытия ошибок CORS в Консоли {: #hide-cors-errors }
 
 <!-- You can hide CORS errors in the **Console**. As the CORS errors are now reported in the Issues tab, hiding CORS errors in the **Console** can help reduce the clutters. -->
+Вы можете скрыть ошибки CORS в **Консоли** (Console). Поскольку ошибки CORS теперь показываются в панели Проблемы (Issues), их скрытие в панели **Консоль** (Console) поможет уменьшить беспорядок.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-96/index.md:177]:
```suggestion
Вы можете скрыть ошибки CORS в **Консоли** (Console). Поскольку ошибки CORS теперь показываются в панели Проблемы (Issues), панель **Консоль** (Console) выглядит менее загроможденной.
```

File: site/ru/blog/new-in-devtools-96/index.md [hunk 10]
```diff
@@ -29,170 +28,243 @@ draft: true
...
 
 
 <!-- ### Consistent async stack traces {: #async } -->
+### Консистентные асинхронные трассировки стека {: #async }
 
 <!-- DevTools now reports `async` stack traces for `async` functions to be consistent with other async tasks.  -->
+DevTools теперь отображает стек трассировки `async` для асинхронных функций, чтобы они соответствовали другим задачам async. 
 
-{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/wuKo84nrDzbhwCnIVU2n.png", alt="async stack traces", width="800", height="427" %}
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/wuKo84nrDzbhwCnIVU2n.png", alt="стек трассировки async", width="800", height="427" %}
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-96/index.md:209]:
```suggestion
{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/wuKo84nrDzbhwCnIVU2n.png", alt="Стек трассировки async", width="800", height="427" %}
```

File: site/ru/blog/new-in-devtools-96/index.md [hunk 11]
```diff
@@ -29,170 +28,243 @@ draft: true
...
 
 
 <!-- ### Retain the Console sidebar {: #console-sidebar } -->
+### Сохранение боковой колонки в Консоли {: #console-sidebar }
 
 <!-- The Console sidebar is here to stay. In Chrome 94, we announced the [upcoming deprecation of the Console sidebar](/blog/new-in-devtools-94/#deprecated) and ask developers for feedback and concerns. -->
+Боковая колонка в Консоли (Console) остается. В Chrome 94 мы анонсировали предстоящее [упразднение боковой колонки в Консоли](/blog/new-in-devtools-94/#deprecated) и попросили разработчиков высказать свои размышления и сомнения по этому поводу.
 
 <!-- We have now got enough feedback from the deprecation notice and we will work on improving the sidebar rather than removing it. -->
+Мы получили достаточное количество отзывов, и теперь будем работать над улучшением боковой панели, а не над её удалением.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-96/index.md:223]:
```suggestion
Мы получили достаточное количество отзывов и теперь будем работать над улучшением боковой панели, а не над её удалением.
```

File: site/ru/blog/new-in-devtools-96/index.md [hunk 12]
```diff
@@ -29,170 +28,243 @@ draft: true
...
 
-Задачи в трекере Chromium: [1254259](https://crbug.com/1254259)
+Задача в трекере Chromium: [1254259](https://crbug.com/1254259)
 
 
 <!-- ### Retain the Console sidebar {: #console-sidebar } -->
+### Сохранение боковой колонки в Консоли {: #console-sidebar }
 
 <!-- The Console sidebar is here to stay. In Chrome 94, we announced the [upcoming deprecation of the Console sidebar](/blog/new-in-devtools-94/#deprecated) and ask developers for feedback and concerns. -->
+Боковая колонка в Консоли (Console) остается. В Chrome 94 мы анонсировали предстоящее [упразднение боковой колонки в Консоли](/blog/new-in-devtools-94/#deprecated) и попросили разработчиков высказать свои размышления и сомнения по этому поводу.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-96/index.md:220]:
```suggestion
Боковая колонка в Консоли (Console) остается на своем месте. В Chrome 94 мы анонсировали предстоящее [упразднение боковой колонки в Консоли](/blog/new-in-devtools-94/#deprecated) и попросили разработчиков высказать свои размышления и сомнения по этому поводу.
```

File: site/ru/blog/new-in-devtools-96/index.md [hunk 13]
```diff
@@ -29,170 +28,243 @@ draft: true
...
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/b0650096c934bf60c21d51ae8a51c94e8f907d38 #}
 
 Задачи в трекере Chromium: [1232937](https://crbug.com/1232937), [1255586](https://crbug.com/1255586)
 
 
 <!-- ## Deprecated Application cache pane in the Application panel {: #app-cache } -->
+## Устаревшая панель Кэш приложения на вкладке Приложение {: #app-cache }
 
 <!-- The [Application cache](https://developer.chrome.com/docs/devtools/storage/applicationcache/) pane in the Application panel is now removed as the support for [AppCache](https://web.dev/appcache-removal/) is removed from Chrome and other Chromium-based browsers. -->
+Панель [Кэш приложения](https://developer.chrome.com/docs/devtools/storage/applicationcache/) (Application cache) на вкладке Приложение (Application) удалена, поскольку [AppCache](https://web.dev/appcache-removal/) больше не поддерживается в Chrome и других браузеров на основе Chromium.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-96/index.md:236]:
```suggestion
Панель [Кеш приложений](https://developer.chrome.com/docs/devtools/storage/applicationcache/) (Application cache) на вкладке Приложение (Application) удалена, поскольку [AppCache](https://web.dev/appcache-removal/) больше не поддерживается в Chrome и других браузерах на основе Chromium.
```
Кеш приложений - именно так это называется в DevTools на русском

File: site/ru/blog/new-in-devtools-96/index.md [hunk 14]
```diff
@@ -29,170 +28,243 @@ draft: true
...
-Задачи в трекере Chromium: [1084190](https://crbug.com/1084190) 
+Задача в трекере Chromium: [1084190](https://crbug.com/1084190) 
 
 
 <!-- ## [Experimental] New Reporting API pane in the Application panel {: #reporting-api } -->
+## [Эксперимент] Новая панель Reporting API на вкладке Приложение {: #reporting-api }
 
 {% Aside %}
 <!-- To enable the experiment, check the **Enable Reporting API panel in the Application panel** checkbox under **Settings** > **Experiments**. -->
+Чтобы включить эксперимент поставьте галочку в чекбоксе напротив пункта **Enable Reporting API panel in the Application panel** во вкладке **Настройки** (Settings) > **Экспериментальные функции** (Experiments).
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-96/index.md:248]:
```suggestion
Чтобы включить эксперимент, поставьте галочку в чекбоксе напротив пункта **Enable Reporting API panel in the Application panel** во вкладке **Настройки** (Settings) > **Экспериментальные функции** (Experiments).
```

File: site/_includes/partials/devtools/ru/whats-new.md [hunk 15]
```diff
@@ -3,18 +3,20 @@
...
-* [Improved the display of properties in the Properties pane](/blog/new-in-devtools-96/#properties)
-* [Option to hide CORS errors in the Console](/blog/new-in-devtools-96/#hide-cors-errors)
-* [Proper `Intl` objects preview and evaluation in the Console](/blog/new-in-devtools-96/#intl)
-* [Consistent async stack traces](/blog/new-in-devtools-96/#async)
-* [Retain the Console sidebar](/blog/new-in-devtools-96/#console-sidebar)
-* [[Experimental] New Reporting API pane in the Application panel](/blog/new-in-devtools-96/#reporting-api) -->
+### Chrome 96 {: #chrome96 }
+* [Ранний доступ: Новая вкладка Обзор CSS](/ru/blog/new-in-devtools-96/#css-overview)
+* [Восстановлен и улучшен опыт редактирования и копирования длины в CSS](/ru/blog/new-in-devtools-966/#length)
+* [Эмуляция CSS-медиафичи prefers-contrast](/ru/blog/new-in-devtools-96/#prefers-contrast)
```
Comment (webmaxru) [inline site/_includes/partials/devtools/ru/whats-new.md:9]:
```suggestion
* [Эмуляция CSS-медиафункции prefers-contrast](/ru/blog/new-in-devtools-96/#prefers-contrast)
```

File: site/ru/blog/new-in-devtools-96/index.md [hunk 16]
```diff
@@ -29,170 +28,243 @@ draft: true
...
+## Восстановлен и улучшен опыт редактирования и копирования длины в CSS {: #length }
+
+<!-- The **copy CSS** and **edit as text** experience are restored for CSS properties with length. These experiences are broken in the last release. -->
+Восстановлена возможность **копировать CSS** и **исправлять как текст** для CSS-свойств с единицами измерения длины. В последнем релизе эти возможности были повреждены.
+
+<!-- {% Video src="video/dPDCek3EhZgLQPGtEG3y0fTn4v82/3zxmVrRNd767L9zPDvU8.mp4", autoplay="true", muted="true", loop="true", class="screenshot" %} -->
+{% Video src="video/dPDCek3EhZgLQPGtEG3y0fTn4v82/3zxmVrRNd767L9zPDvU8.mp4", autoplay="true", muted="true", loop="true", class="screenshot" %}
+
+<!-- In addition, you can drag to adjust the unit value and update the unit type via the dropdown. This add-on length authoring feature should not impact the primary edit as text experience. -->
+Кроме того, можно перетаскивать для настройки значения единицы измерения и обновлять тип единицы измерения с помощью выпадающего списка. Эта дополнительная функция редактирования длины не должна влиять на основное редактирование в виде текста.
```
Comment (webmaxru) [inline site/ru/blog/new-in-devtools-96/index.md:65]:
```suggestion
Кроме того, значение можно изменять перетаскиванием, а единицы измерения задавать с помощью выпадающего списка. Эта дополнительная возможность редактирования не влияет на основное редактирование в виде текста.
```

**Review by webmaxru — CHANGES_REQUESTED** [review]

**Review by chybie — APPROVED** [review]

**Comment by chybie** [comment]
Спасибо вам всем!

**Pull request merged**